lesson08/assignment/video_cli/l8/src/App.vue

```diff
@@ -3,19 +3,17 @@
     <Navbar />
     <router-view />
     <ContactForm />
-    <post
-    v-for="post in posts"
-    v-bind:post="post"
-    v-bind:key="post.id">
+    <post v-for="(post, id) in posts"
+      :post="post"
+      :key="id">
+  </post>
   </div>
 </template>
 
 <script>
 //imports pages
 import Navbar from './Navbar.vue';
-import Home from '/Home.vue';
```

Comment:
Home and Top3 components don't need to be imported here.  They are imported in main.js where the router uses them to create the routes.